x/auth/client/testutil/suite.go

```diff
@@ -1460,6 +1464,299 @@ func (s *IntegrationTestSuite) TestSignWithMultiSignersAminoJSON() {
 	require.Equal(sdk.NewCoins(val0Coin, val1Coin), queryRes.Balances)
 }
 
+func (s *IntegrationTestSuite) TestTipsWithFee() {
```

Comment:
I think we don't deal with fees in this test

```suggestion
func (s *IntegrationTestSuite) TestAuxSigner() {
```